packages/ckeditor5-dev-utils/lib/bundler/getlicensebanner.js

```diff
@@ -10,10 +10,10 @@ module.exports = function getLicenseBanner() {
 
 	/* eslint-disable indent */
 	return (
-`/**
- * @license Copyright (c) 2003-${ date.getFullYear() }, CKSource - Frederico Knabben. All rights reserved.
- * For licensing, see LICENSE.md.
- */`
+`/*--------------------------------------------------------------------------------------*
```

Comment:
It doesn't seem like nicely aligned. Is it GH's way of rendering  this string? Or isn't the box wide enough?

Reply:
`${ date.getFullYear() }` that becomes 4-character-long as it evaluates.